services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/component/DccRulesNotFetchedStructureProviderTest.java

```diff
@@ -151,9 +151,9 @@ void shouldContainEmptyInvalidationRules() throws FetchBusinessRulesException {
     businessRuleItem2.setCountry("test2");
 
     BusinessRule businessRule = new BusinessRule();
-    businessRule.setType(RuleType.Acceptance.name());
+    businessRule.setType(RuleType.ACCEPTANCE.name());
     BusinessRule businessRule2 = new BusinessRule();
-    businessRule2.setType(RuleType.Acceptance.name());
+    businessRule2.setType(RuleType.ACCEPTANCE.name());
 
```

Comment:
same here. The test is working because the comparison is done using ignoreCase when creating the files. But a similar test for BOOSTER_NOTIFICATION will fail.